### PR TITLE
Added whoami alias

### PR DESCRIPTION
**Pull request by bvulaj**
```
$ uhc whoami
{
  "id": "1MpVidk1UTdrdD8SGzXpHrAOR4b",
  "kind": "Account",
  "href": "/api/accounts_mgmt/v1/accounts/1MpVidk1UTdrdD8SGzXpHrAOR4b",
  "first_name": "Brandon",
  "last_name": "Vulaj",
  "username": "bvulaj.openshift",
  "email": "bvulaj@redhat.com",
  "banned": false,
  "created_at": "2019-06-18T18:14:13.670937Z",
  "updated_at": "2019-06-18T18:15:04.747902Z",
  "organization": {
    "id": "1MKVU4otCIuogoLtgtyU6wajxjW",
    "kind": "Organization",
    "href": "/api/accounts_mgmt/v1/organizations/1MKVU4otCIuogoLtgtyU6wajxjW",
    "name": "Red Hat : Service Delivery : SDA/B"
  }
}
```

### CODE REVIEW

File: cmd/uhc/whoami/cmd.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
```
Comment (jhernand) [inline cmd/uhc/whoami/cmd.go]: 2019

File: cmd/uhc/whoami/cmd.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/dump"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+
+	"github.com/spf13/cobra"
```
Comment (jhernand) [inline cmd/uhc/whoami/cmd.go]: The order for imports in the project is first the standard library, then the third party packages, and then the project packages. That means the following:

```go

import (
	"fmt"
	"os"

	"github.com/openshift-online/uhc-sdk-go/pkg/client"
	"github.com/spf13/cobra"

	"github.com/openshift-online/uhc-cli/pkg/config"
	"github.com/openshift-online/uhc-cli/pkg/dump"
	"github.com/openshift-online/uhc-cli/pkg/util"
}
```

Can you do that change?

**Review by jhernand — COMMENTED** [review]
Thanks @bvulaj, looks good. I have however a couple of very minor comments. Address them and squash to one commit. I will then merge.

File: cmd/uhc/whoami/cmd.go (hunk 1 above)
Comment (bvulaj) [reply cmd/uhc/whoami/cmd.go]: Fixed.

File: cmd/uhc/whoami/cmd.go (hunk 2 above)
Comment (bvulaj) [reply cmd/uhc/whoami/cmd.go]: Fixed

**Comment by bvulaj** [comment]
Fixed, rebased, and squashed.

**Review by jhernand — APPROVED** [review]
Looks good, thanks @bvulaj.

**Pull request merged**